docs/reference/mapping/types/geo-point.asciidoc

```diff
@@ -8,9 +8,12 @@ Fields of type `geo_point` accept latitude-longitude pairs, which can be used:
 
 * to find geopoints within a <<query-dsl-geo-bounding-box-query,bounding box>>,
   within a certain <<query-dsl-geo-distance-query,distance>> of a central point,
-  or within a <<query-dsl-geo-polygon-query,polygon>> or within a <<query-dsl-geo-shape-query,`geo_shape` query>>.
-* to aggregate documents <<search-aggregations-bucket-geohashgrid-aggregation,geographically>>
-  or by <<search-aggregations-bucket-geodistance-aggregation,distance>> from a central point.
+  or within a <<query-dsl-geo-shape-query,`geo_shape` query>> (for example, points in polygon).
```

Comment:
```suggestion
  or within a <<query-dsl-geo-shape-query,`geo_shape` query>> (for example, points in a polygon).
```

Reply:
I think `point in polygon` is ok in this context as it is the given name for a specific geo problem